godtools/push/action.yaml

```
@@ -14,14 +14,18 @@ inputs:
     required: false
     default: acr
     description: List of names of the registries where an image should be pushed (ecr, acr, docker-hub, internal-acr).
+  tag-latest:
+    required: false
+    default: "true"
+    description: Prepare desired tag from latest image before pushing it to the registry
 runs:
   using: "composite"
   steps:
     - name: Push tmp docker image
       if: "!startsWith(github.ref, 'refs/tags')"
       shell: bash
       run: |
-        docker tag ${{ inputs.image }} ${{ inputs.image }}:tmp-$(git rev-parse --short HEAD) && 
+        ["${{ inputs.tag-latest }}" = "true"] && docker tag ${{ inputs.image }} ${{ inputs.image }}:tmp-$(git rev-parse --short HEAD); 
```

Currently running this action returns unknown command
```
Run PiwikPRO/actions/godtools/push@feature/PPPSYS-42560-tag-latest-flag
Run ["false" = "true"] && docker tag cdp/reminder cdp/reminder:1.57.1-push-godtools-test-2;
/home/runner/work/_temp/12a[8](https://github.com/PiwikPRO/PPMS-CDP/actions/runs/5786460552/job/15681220031#step:11:9)07bb-fffa-4f77-b986-bbe3eb88c1d9.sh: line 1: [false: command not found
```


I think that needs a whitespace between `[` and `"`. 38th line as well 
```suggestion
        [ "${{ inputs.tag-latest }}" = "true" ] && docker tag ${{ inputs.image }} ${{ inputs.image }}:tmp-$(git rev-parse --short HEAD);
```

---

You're right! my bad.

---

Tested out on organization service with both configurations - works fine now :ok_hand: 

[enabled](https://github.com/PiwikPRO/PPMS-Organization/actions/runs/5793195842/job/15700578636#step:8:11) / [disabled](https://github.com/PiwikPRO/PPMS-Organization/actions/runs/5793216112/job/15700631285#step:8:11)

disabled fails as expected since latest is not tagged.